src/__main__.py

```diff
@@ -69,7 +70,10 @@ def excepthook(exc_type, exc_value, traceback_object):
         raise exc_value
 
     logger.error("Uncaught exception", exc_info=(exc_type, exc_value, traceback_object))
-    logger.error("Runtime Info:\n%s", util.runtime_info())
+    runtime_info = util.runtime_info()
+    logger.error("Runtime Info:\n%s", runtime_info)
+    bugsnag.notify(exc_value, meta_data={"runtime_info": runtime_info})
```

Comment:
Can this throw any exceptions, e.g. when internet connection is missing? We should catch everything that's not catastrophic within the excepthook so that it's at least shown properly.